### PR TITLE
Rubberband guide 1178

### PR DESCRIPTION
**Pull request by teresam856**
In this PR:
[Issue 1178](https://github.com/GMOD/jbrowse-components/issues/1178)

RubberBand.tsx
- modified vertical guide to include refname
- need to add start to the guide px offset to fix the bug

Without refName
ruuberband_guide

withRefName
rubberband_guide_with_refname

### CODE REVIEW

**Comment by cmdcolin** [comment]
Nice start...can you test with the display horizontally flipped, see screenshot. I think it ends up that just adding start to the offset ends up not working in this case

t111

**Comment by teresam856** [comment]
@cmdcolin Got it, I see the issue when flipping it horizontally. Will look into that.

File: plugins/linear-genome-view/src/LinearGenomeView/components/OverviewRubberBand.tsx [hunk 1]
```diff
@@ -164,10 +164,12 @@ function OverviewRubberBand({
           <Tooltip
             open={!mouseDragging}
             placement="top"
-            title={Math.max(
+            title={`${overview
+              .pxToBp(guideX)
+              .refName.toLocaleString()} ${Math.max(
```
Comment (cmdcolin) [inline plugins/linear-genome-view/src/LinearGenomeView/components/OverviewRubberBand.tsx:169]: refName probably doesn't need toLocaleString

Comment (cmdcolin) [reply plugins/linear-genome-view/src/LinearGenomeView/components/OverviewRubberBand.tsx:169]: (toLocaleString is only needed for numbers, it adds commas to long numbers, etc. Note that any use of this should use value.toLocaleString('en-US'))

That is using the US system of comma-fying long numbers which is US biased but it's just how the rest of the app displays numbers now

**Comment by cmdcolin** [comment]
This looks good. I pushed a couple commits to the branch to possibly help

Can you review these two commits and retest and see if this still is ok?

**Comment by teresam856** [comment]
@cmdcolin sure thing

**Comment by cmdcolin** [comment]
this looks good to me now, want to go ahead and merge? should be able to fix the dotplot flaky separately

**Pull request merged**